r2/r2/lib/jsontemplates.py

```diff
@@ -397,6 +398,12 @@ def thing_attr(self, thing, attr):
                 return c.user.use_subreddit_style(thing)
             else:
                 return True
+        elif attr == 'author':
+            if not thing.author:
```

Comment:
Normally, this `elif` block wouldn't need to be added, and the subreddit creator's name would be retrieved via the `else` block. However, there are times that there is *no* author (or at least that seems to be the case from the SubredditInfoBar `creator_text` [double checking if the subreddit has an author](https://github.com/reddit/reddit/blob/c93f7b8c84895bf5c5df93516e1b47cb315484e4/r2/r2/lib/pages/pages.py#L1177) in the firstplace, and /r/askreddit not having a creator, so it isn't showing up because of [this similar check](https://github.com/reddit/reddit/blob/85f61036814a3f58ccdb4a4d85d83ce07c292425/r2/r2/templates/subredditinfobar.html#L109), just as an example), and unless I'm mistaken, the `ThingJsonTemplate`'s `thing_attr` does not have a check for items that have no author, and when the `name` attribute will be called on the `None` that the author would be, you'd get an AttributeError.